designer/server/views/designer.html

```diff
@@ -9,6 +9,19 @@
 {% endblock %}
 
 {% block content %}
+    {% if (phase) %}
+    <div class="govuk-phase-banner">
+      <p class="govuk-phase-banner__content">
+        <strong class="govuk-tag govuk-phase-banner__content__tag">
+        {{ phase }}
+        </strong>
+        <span class="govuk-phase-banner__text">
+        This service is currently in development
```

Comment:
i18n string here?

Reply:
this is one of the njk templates.. we don't have i18n hooked up to this (if at all)..!

Reply:
👍